website/docs/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith.md

```diff
@@ -98,57 +130,48 @@ async def root():
 
-Awesome, with this we should see our API running at [http://127.0.0.1:8000](https://github.com/appsmithorg/appsmith-docs/blob/v1.3/how-to-guides/http:/127.0.0.1:8000!) !
+You would see a screen below when the app is up and API is ready to use.
 
-### Setting ngrok
+![Run the localhost API using FastAPI](/img/start-localhost-api-using-fastapi.png)
 
-Now that we have our local APIs, let’s use `ngrok` to serve them on production. For this, we’ll have to signup at [ngrok](https://dashboard.ngrok.com/get-started/setup) (it’s free!), and follow the instructions to connect your account.
+Awesome, you could see your [API in action](http://127.0.0.1:8000).
 
-1. Download the `ngrok` installation file and unzip it.
-2. Add your `auth-token` to the default ngrok.yml configuration file using:
+Fire up `ngrok` and expose the local port[8000] to access the local API in your app available on [Appsmith Cloud](https://app.appsmith.com).
 
-```
-$ ./ngrok authtoken <your-auth-token>
+```bash
+ngrok <PROTOCOL> <LOCAL_PORT> 
 ```
 
-Fire up `ngrok`, we already have our API server ready, now we’ll have to expose the particular local port for HTTP tunnel forwarding using the following command:
+To access the API, you would have to use the `HTTP` protocol and port `8000`.
 
+```bash
+ngrok http 8000
 ```
-$ ./ngrok http 8000
-```
-
-Awesome, we can now see that the local APIs are now being forwarded to a different server on production using `ngrok`. Below is a screenshot,
 
-![ngrok commands](https://lh5.googleusercontent.com/5Qdqw3U5EYtDk5EhpWrTrrUw5EcKPqZGE8xX2W7NjazBd\_cdSQZNUgVkUzkQXjG0NqCusqQW4ftUp3GAOg794gsWCZpXrDi1lmtBF7ZplJ5lAAcdsc\_hfKOFr93KebVE4nZ1JKp9)
+`ngrok` creates an HTTP tunnel, forwards the externally accessible address to the local address, and enables access to the local API over the internet.
 
-### Testing APIs on Appsmith
+![connect local api using ngrok](/img/connect-localhost-api-using-ngrok.png)
 
-Now that we have API, up and running, let’s test these out on Appsmith.
-
-* Create an application on Appsmith
-* Create a New API by clicking on the `+` icon next to the APIs section on the left navigation and rename it to `getGames`
-* Add the forwarded API link and hit RUN, we’ll have to see the API response on the response pane.
-* Next, navigate to Pages, and drag and drop a Table Widget onto the canvas.
-* Open the table property pane and add use the API to display data on to the table by adding the following code snippet under the Table Data property:
-
-```javascript
-{{ getGames.data }}
-```
+#### Create API
+You can [create and API](core-concepts/connecting-to-data-sources/authentication/connect-to-apis#api-editor) and add the localhost API with the address `https://a8cc-2405-201-21-4011-5564-59ac-2209-1c4c.in.ngrok.io`.
 
-Below is a GIF, following the same steps:
+![connect to an API hosted on localhost using ngrok](/img/create-api-for-local-api-on-appsmith.png)
 
-![Test API](/img/NGROK.gif)
+## Displaying data
```

Comment:
Is this supposed to be a subheading of the "### Building a simple local API" section? This is currently a 2nd (##) level heading on the same level as "# ngrok" at the top of the doc

Reply:
Yes. It's more like, 'Whats next?' Should we call it 'next steps' if that makes more sense? What do you think?

Reply:
I think that might be good, I like the idea of updating it to "Next steps".

Reply:
Done.